_1327/documents/migrations/0011_auto_20180827_1844.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-27 16:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+	dependencies = [
+		('documents', '0010_add_on_delete'),
+	]
+
+	operations = [
+		# migrations.RemoveField(
+		#     model_name='document',
+		#     name='text',
+		# ),
+		# migrations.AddField(
+		#     model_name='document',
+		#     name='text_de',
+		#     field=models.TextField(blank=True, null=True),
+		# ),
+		migrations.AlterField(
+			model_name='document',
+			name='text',
+			field=models.TextField(blank=True, null=True),
+		),
+		migrations.RenameField(
+			model_name='document',
+			old_name='text',
+			new_name='text_de',
+		),
+		# migrations.RemoveField(
+		#     model_name='document',
+		#     name='title',
+		# ),
+		# migrations.AddField(
+		#     model_name='document',
+		#     name='title_de',
+		#     field=models.CharField(blank=True, max_length=255, null=True),
+		# ),
+		migrations.AlterField(
+			model_name='document',
+			name='title',
+			field=models.CharField(blank=True, max_length=255, null=True),
```

Comment:
Yes, because it will be set based on the translation of title_de/title_en. Ths way it can be null initially.

Reply:
maybe we're talking past each other, i was asking because i assumed a document *always* needs a title, or after your changes, both an english and a german title. what do you mean by "will be set basedon the translation of title_de/en"?

Reply:
My idea was that a document always needs a German title. The English one is optional. The `title` field is automatically set by the following line in documents/models.py:
`title = translate(en='title_en', de='title_de')`
I assumed that this way title will be null initially and the reason why `null=True`.
I hope this answers the question.

Reply:
to avoid any misunderstanding, `title` is not a database field at all (after your migrations are applied, that is), just a property that either looks into `title_de` or `title_en`, but that's done in python and not at the database level. a few lines below, you're renaming `title` to `title_de`, and after that there's no `title` field in the database anymore.

if you want the german title to be mandatory, that's not what the code in your PR says right now as far as i can see, it says `null=True` both in the model and the migration.